Output/UARTOut/arm/uart_serial.c

```diff
@@ -100,6 +122,8 @@ volatile uint8_t uart_configured = 0;
 void uart0_status_isr()
 #elif defined(_kii_v2_) // UART2 Debug
 void uart2_status_isr()
+#elif defined(_kii_v3_) //UART? Debug
+void uart0_status_isr()
```

Comment:
Look at the pin mapping spreadsheet. This is USART1. So USART1_Handler().

Reply:
USART or UART?

Reply:
On the sam4s there's USART0 USART1 UART0 and UART1

In this case USART1